sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/ADTCollection.cs

```diff
@@ -15,11 +11,11 @@ public enum AdtCollection
         /// <summary>
         /// DigitalTwins collection in an ADT instance.
         /// </summary>
-        DigitalTwins = 0,
+        DIGITALTWINS = 0,
```

Comment:
Q: What happens if your digital twins were formatted as it was previously? does the query explorer complain?

Reply:
Usually, Enums are not named with all-caps in .NET

Reply:
The query is still valid, the tests I were running just failed since I was comparing DigitalTwins to DIGITALTWINS. I don't mind changing it back if it seems to be easier to look at or something, it was just the quicker fix for the error.

Reply:
> Usually, Enums are not named with all-caps in .NET

okay, maybe I should just change the tests then.